docs/fleet/machines/control.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Control Machines"
+linkTitle: "Control"
+weight: 10
+type: "docs"
+description: "A machine is an organizational concept, consisting of either one or multiple parts working closely together to complete tasks."
+tags: ["fleet management", "control", "app"]
+images: ["/fleet/control.gif"]
+---
+
+Once you have configured components and services for your machine, you can test, monitor, and remotely operate them from the **CONTROL** tab in the [Viam app](https://app.viam.com) or the [Viam mobile app](/fleet/#the-viam-mobile-app).
+
+## Control interface in the Viam app
+
+The **CONTROL** tab in the [Viam app](https://app.viam.com) gives you the ability to test, monitor, and operate the machines in your fleet.
+The **CONTROL** tab provides a control interface for each component and service that you have configured for you machine.
+
+For example, if you have configured a base with wheels, you can move your machine's with an arrow pad and control the base's speed by setting its power with a slider.
+If you have configured a camera component, a window in the **CONTROL** tab displays the camera output.
+
+If you use remote control in the [Viam app](https://app.viam.com) UI, all communication to the machine uses [WebRTC](https://pkg.go.dev/go.viam.com/utils@v0.0.3/rpc#hdr-Connection).
+For local communication between [parts](/build/configure/parts/#machine-parts) Viam uses gRPC or WebRTC.
+
+{{<gif webm_src="/fleet/control.webm" mp4_src="/fleet/control.mp4" alt="Using the control tab" max-width="800px">}}
+
+### Components
+
+For more detailed information on how to operate and test your resources, expand the relevant resource below:
+
+{{% expand "arm" %}}
```

Comment:
I think all component names should be capitalized (probably sentence case, not title case)
```suggestion
{{% expand "Arm" %}}
```